src/build.props

```diff
@@ -42,7 +42,7 @@
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" Condition="'$(DisableBinSkimTarget)'!='true'"/>
   <PropertyGroup>
     <BinSkimVerboseOutput>false</BinSkimVerboseOutput>
-    <BinSkimExePath>$(MsBuildThisFileDirectory)packages\Microsoft.CodeAnalysis.BinSkim.1.3.9\tools\x64\BinSkim.exe</BinSkimExePath>
+    <BinSkimExePath>$(MsBuildThisFileDirectory)packages\Microsoft.CodeAnalysis.BinSkim.1.4.0\tools\x64\BinSkim.exe</BinSkimExePath>
   </PropertyGroup>
```

Comment:
I never noticed this before: why does the SARIF SDK have a dependency on (or at least, refer to) a particular tool built from the SDK?